docker/entrypoint_update.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -l
```

Comment:
the bash files need to be executable (chmod 711 *.sh)

Reply:
[a4d472a](https://github.com/ElsLommelen/protocolsource/pull/6/commits/a4d472ad211cd3cf1da9a82c07a5ee847b5b5b4a)